docs/ruleset_functions/get_HVAC_systems_applicable_10cd.md

```diff
@@ -0,0 +1,82 @@
+## get_HVAC_systems_applicable_10cd
+
+**Description:** Get the list of HeatingVentilationAirAconditioningSystem in which Appendix G Table G3.1 #10 c and d are applicable.
+
+**Inputs:**
+- **U-RMR**: The U-RMR to determine if an HVAC system has been designed or modeled as existing for each zone.
+- **P-RMR**: To determine if the same zone has been analyzed with a HVAC system in P-RMR.
+
+**Returns:**
+- **applicable_hvac_systems_10cd_list_p**: A list that saves all the HVAC systems in which Appendix G Table G3.1 #10 c and d are applicable.
+ 
+**Function Call:** 
+
+1. match_data_element()
+
+**Logic:**
+- # Determine if a HVAC system has been designed or modeled as existing in U-RMR for each zone**
```

Comment:
I imagined this function would compare HVAC systems in the U-RMR and P-RMR and identify those that have cooling or heating component in U but not P. Perhaps it also needs to check if there are heating/cooling associated with terminals (via the existing heat_capacity field and cool_capacity field to be added). But it seems to be doing much more than that. Let's discuss the intended logic on the call.